ops/jenkins/bfd-run-synthea-generation/Jenkinsfile

```diff
@@ -51,6 +51,17 @@ spec:
            description: 'The number of months into the future that claim lines should be '
                         + 'generated. If specified, future claims will be split in the '
                         + 'generated output to allow for proper loading')
+                        
+    booleanParam(
+        name: 'USE_TARGET_CONTRACT',
+        description: 'If checked, use the below contract partD contract number for all generated items.',
+        defaultValue: true
+    )
+                        
+    string(name: 'TARGET_CONTRACT',
+           defaultValue: 'Y9999',
+           description: 'If USE_TARGET_CONTRACT is checked, indicates a single part D contract that all generated claims'
+                        + ' will be associated with.')
```

Comment:
As a design aside, I went with two fields, one for the value and one for verifying to use the contract, for the following reasons:
- Avoids jenkins logic to only add the param if some value is present in the box
- Avoids having to worry about trying to call the script with a null jenkins-passed value or worry about catching that case (which generally makes the script just not run)
- Avoids having an optional param (which might make extension harder at the prepare-and-run-synthea script level, since it just uses a flat list of args, and making one optional means having to parse them out individually instead of just using an index)